index.html

```diff
@@ -1,25 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!--awesome font to get the icons -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">    <!--custom made css -->
-    <link rel="stylesheet" href="css/style.css">    
```

Comment:
- [ ] You can see that in the Figma template the close icon is a little bit higher than menu links **(Portfolio, About, Contact)** Kindly make your close icon a little bit higher from the current position :blush:

- [OPTIONAL] Great job so far :raised_hands: but I think it would be better if you could use a `position: fixed;`  instead of absolute. Fixed position prevents your navigation page menu page from scrolling. The navigation is always fixed to the page instead of scrolling. Please change this. This will make your website similar to the original design. 💯 🥇 

- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 
 
- [OPTIONAL] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 





###   Adding these features will make your design look great and look exactly like the Figma design, I'll make this as an optional requiement since the previous reviewer is skipping this but I highly recommend you do this in the meantime, 


| Your design | Figma design |
| --- | --- |
| ![Screenshot from 2022-10-11 02-15-16](https://user-images.githubusercontent.com/79058364/194970996-77ea4423-9964-49d5-bb1f-c8bc91d184e7.png) | ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |